File: app/src/main/java/cn/intret/app/picgo/model/ImageGroup.java
package cn.intret.app.picgo.model;

import java.util.Date;
import java.util.List;

/**
 * Image Group
 */

public class ImageGroup {

    Date mStartDate;
    Date mEndDate;
    List<MediaFile> mMediaFiles;

    public Date getStartDate() {
        return mStartDate;
    }

    public ImageGroup setStartDate(Date startDate) {
        mStartDate = startDate;
        return this;
    }

    public Date getEndDate() {
        return mEndDate;
    }

    public ImageGroup setEndDate(Date endDate) {
        mEndDate = endDate;
        return this;
    }

    public List<MediaFile> getMediaFiles() {
        return mMediaFiles;
    }

    public ImageGroup setMediaFiles(List<MediaFile> mediaFiles) {
        mMediaFiles = mediaFiles;
        return this;
    }

    @Override
    public String toString() {
        return "ImageGroup{" +
                "mStartDate=" + mStartDate +
                ", mEndDate=" + mEndDate +
                '}';
    }
}

File: app/src/main/java/cn/intret/app/picgo/model/image/ImageGroup.kt
package cn.intret.app.picgo.model.image

import java.util.Date

/**
 * Image Group
 */

class ImageGroup {

    internal lateinit var startDate: Date
    internal lateinit var endDate: Date
    internal lateinit var mediaFiles: List<MediaFile>

    fun getStartDate(): Date {
        return startDate
    }

    fun setStartDate(startDate: Date): ImageGroup {
        this.startDate = startDate
        return this
    }

    fun getEndDate(): Date {
        return endDate
    }

    fun setEndDate(endDate: Date): ImageGroup {
        this.endDate = endDate
        return this
    }

    fun getMediaFiles(): List<MediaFile> {
        return mediaFiles
    }

    fun setMediaFiles(mediaFiles: List<MediaFile>): ImageGroup {
        this.mediaFiles = mediaFiles
        return this
    }

    override fun toString(): String {
        return "ImageGroup{" +
                "startDate=" + startDate +
                ", endDate=" + endDate +
                '}'.toString()
    }
}
